ts/create-kpt-functions/src/cmd/type_create.ts

```diff
@@ -87,6 +67,73 @@ export async function typeCreate(packageDir: string) {
   log(format.finishMarker(desc));
 }
 
+// chooseTypeSource lets the user choose the source for the kubernetes types and returns
+// numeric choice
+function chooseTypeSource(): number {
+  const defaultContext = 0;
+  const ch = validator.getValidString(
+    () =>
+      question(
+        `> Kubernetes Types used to generate Typescript library (0) \n` +
```

Comment:
Simpler to do this:

`Multi line
Do not close
until the
end`

Reply:
Thanks. Done.